### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by JamesFrierson1**
Added additional reference and license information to the README per #49 

For some reason I was unable to hyperlink human.brain-map.org but that just may because I'm new at this! Please let me know what else I can do 👍

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -79,6 +79,10 @@ code used, and (2) cite their paper:
    guide to linking brain-wide gene expression and neuroimaging data. bioRxiv,
    380089.
 
+Additionally, please add the following publication to all Acknowledgments and References sections per The Allen Institute's `citation policies <https://alleninstitute.org/legal/citation-policy/>`_:
```
Comment (rmarkello) [inline README.rst]:
```suggestion
Additionally, please cite the following paper as the source of all microarray expression data from the Allen Human Brain Atlas:
```

**Comment by JamesFrierson1** [comment]
Sounds great, @rmarkello !

**Comment by rmarkello** [comment]
Hey @JamesFrierson1&mdash;I think you just marked the suggestion as resolved rather than committing the changes! You should be able to [click the "commit suggestion"](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request) button to incorporate it. Once you do that I'll go ahead and merge :smile:

**Comment by rmarkello** [comment]
Perfect! :tada: Once the CI passes I'll go ahead and merge.

Thanks so much for your contribution!

**Comment by JamesFrierson1** [comment]
Thanks, @rmarkello ! Appreciate your patience 😄

**Pull request merged**